File: Tareas_FdP/Tarea02vscode/Ejercicio08.cs
using System;

class Cilindro {
    static void Main() {
        double radio, altura, area, volumen;

        Console.WriteLine("Ingrese el radio del cilindro: ");
        radio = Convert.ToDouble(Console.ReadLine());
        Console.WriteLine("Ingrese la altura del cilindro: ");
        altura = Convert.ToDouble(Console.ReadLine());

        area = 2 * Math.PI * radio * (radio + altura);
        volumen = Math.PI * Math.Pow(radio, 2) * altura;

        Console.WriteLine("El área del cilindro es: " + area);
        Console.WriteLine("El volumen del cilindro es: " + volumen);
    }
}

File: Tareas_FdP/Tarea02vscode/Ejercicio08.py
import math

print("Ingrese el radio del cilindro: ")
radio = float(input())

print("Ingrese la altura del cilindro: ")
altura = float(input())

area = 2 * math.pi * radio * (radio + altura)
volumen = math.pi * (radio ** 2) * altura

print("El área del cilindro es: ", area)
print("El volumen del cilindro es: ", volumen)